app/controllers/application_controller.rb

```diff
@@ -1,5 +1,12 @@
 class ApplicationController < ActionController::Base
-  # Prevent CSRF attacks by raising an exception.
-  # For APIs, you may want to use :null_session instead.
+  before_action :configure_permitted_parameters, if: :devise_controller?
+  
   protect_from_forgery with: :exception
+
+  protected
+
+  def configure_permitted_parameters
+  	temp = [:first_name, :last_name, :email, :password, :password_confirmation]
```

Comment:
nice solution! however I would recommend to rewrite this as
[:first_name, :last_name, :email, :password, :password_confirmation].each do |field|
  devise_parameter_sanitizer.for(:sign_up) << field
end

How do you think? Which syntax do you like best?